spec/index.md

```diff
@@ -305,13 +278,19 @@ Each dimension is represented by a hash in the `dimensions` array. The hash has
     "primaryKey": ["Project", "ClassCode"],
 
     // OPTIONAL: Describes what kind of a dimension it is. `dimensionType` is a string that `MUST` be one of the following:
-    // datetime, entity, classification, activity, fact, location, other`
-    "dimensionType": "classification",
+    // * "datetime": the date of a transaction 
+    // * "entity": names the organisation doing the spending or receiving
+    // * "classification": one or more fields that create a categorical hierarchy of the type of spending (eg, Health > Hospital services > Nursing) 
+    // * "activity": one or more fields that create an administrative hierarchy (eg, Department > Programme > Project)
```

Comment:
Thanks - didn't know they existed. (To be honest, I've been kind of ignoring the later parts of the document - are they actually normative?)

Does my update this address the issue, @danfowler ?